test/jobs/maintenance_tasks/task_job_test.rb

```diff
@@ -131,7 +131,7 @@ class TaskJobTest < ActiveJob::TestCase
 
       run.expects(:persist_error).with do |exception|
         assert_kind_of ArgumentError, exception
-        assert_equal "Something went wrong", exception.message
+        assert_equal "Something went wrong #{"!" * 255}", exception.message
```

Comment:
This message is longer than 255 characters, so what does the assertion check here?

Reply:
It asserts that even if the error is longer than 255 chars the error will be correctly persisted.